doc/source/whatsnew/v2.2.0.rst

```diff
@@ -89,6 +89,40 @@ a Series. (:issue:`55323`)
     )
     series.list[0]
 
+.. _whatsnew_220.enhancements.adbc_support:
+
+ADBC Driver support in to_sql and read_sql
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:func:`read_sql` and :meth:`to_sql` now work with `Apache Arrow ADBC
+<https://arrow.apache.org/adbc/current/index.html>`_ drivers. Compared to
+traditional drivers used via SQLAlchemy, ADBC drivers should provide
+significant performance improvements, better type support and cleaner
+nullability handling.
+
+.. ipython:: python
```

Comment:
You need to make this a literal `.. code-block::`, or otherwise use sqlite in-memory. Because this code is run now, and there is no database named "postgres" set up, I assume?

Reply:
Ah that makes sense thanks